pilot/pkg/networking/core/v1alpha3/listener.go

```diff
@@ -291,7 +291,17 @@ func (lb *ListenerBuilder) buildSidecarOutboundListeners(node *model.Proxy,
 ) []*listener.Listener {
 	noneMode := node.GetInterceptionMode() == model.InterceptionNone
 
-	actualWildcard, actualLocalHostAddress := getActualWildcardAndLocalHost(node)
+	oWildcardAndLocalHost := NewWildcardAndLocalHost(node.GetIPMode())
```

Comment:
Have methods like Wildcards() and Localhosts() on node it self. That simplifies lot of code I guess.

Reply:
Hi @ramaraochavali, changed them into proxy node.